srcpkgs/python3-txredisapi/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-txredisapi'
+pkgname=python3-txredisapi
+version=1.4.7
+revision=1
+wrksrc="txredisapi-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-Twisted python3-six"
+checkdepends="$depends redis python3-pytest python3-mock"
+short_desc="Non-blocking redis client for python"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="Apache-2.0"
+homepage="https://github.com/IlyaSkriblovsky/txredisapi"
+changelog="https://raw.githubusercontent.com/IlyaSkriblovsky/txredisapi/master/CHANGELOG.md"
+distfiles="https://github.com/IlyaSkriblovsky/txredisapi/archive/refs/tags/${version}.tar.gz"
+checksum=dfaf204ab547eeddfcf54c81734a8a06dadbbf079fd01d9c27a788c7d770376e
+
+do_check() {
+	redis-server &
+	python3 -m pytest --ignore tests/test_unix_connection.py
+	redis-cli flushall
+	redis-cli shutdown
+}
```

Comment:
This can unfortunately have unintended consequences - redis-server doesn't get shut down when pytest fails. So don't start redis-server and disable the tests which need it.

Reply:
Sure thing. I can either `--ignore ...` the tests which need it...
```
make_check_args="--ignore tests/test_basics.py
 --ignore tests/test_bitops.py
 --ignore tests/test_blocking.py
 --ignore tests/test_cancel.py
 --ignore tests/test_connection.py
 --ignore tests/test_connection_charset.py
 --ignore tests/test_hash_ops.py
 --ignore tests/test_hyperloglog.py
 --ignore tests/test_implicit_pipelining.py
 --ignore tests/test_list_ops.py
 --ignore tests/test_multibulk.py
 --ignore tests/test_number_conversions.py
 --ignore tests/test_operations.py
 --ignore tests/test_pipelining.py
 --ignore tests/test_publish.py
 --ignore tests/test_scan.py
 --ignore tests/test_scripting.py
 --ignore tests/test_sets.py
 --ignore tests/test_sort.py
 --ignore tests/test_sortedsets.py
 --ignore tests/test_subscriber.py
 --ignore tests/test_transactions.py
 --ignore tests/test_unix_connection.py
 --ignore tests/test_watch.py"
```
Or I can only run the ones that work without it:
```
make_check_target="tests/test_protocol.py tests/test_sentinel.py"
```
`make_check_target` approach is shorter, but I consider the `--ignore` approach to be more correct, so I'll use that unless you disagree.

Reply:
It's ugly either way, I will leave it up to you since you are the maintainer :wink: